### PR TITLE
Improve defaults handling in GUI

### PR DESCRIPTION
**Pull request by zaucker**
- Leave null values in inputs when sending to GUI (was forced to empty string before)
- Show `Standard` in GUI if input value is null and has a default
- Count as "complete" in this case
- Some cleanup of table cell renderers
- Write input validation errors to server log

### CODE REVIEW

File: frontend/source/translation/fr.po [hunk 1]
```diff
@@ -974,3 +974,7 @@ msgstr ""
 #: agrammon/Changelog.js
 msgid "Changelog"
 msgstr ""
+
+#. NO LONGER USED
+msgid "Standard"
+msgstr ""
```
Comment (jnthn) [inline frontend/source/translation/fr.po:980]: If it's no longer used, it can't be removed?

File: lib/Agrammon/DataSource/Util.pm6 [hunk 2]
```diff
@@ -1,5 +1,6 @@
 use v6;
 
 sub maybe-numify($value) is export {
+    return $value if not $value.defined;
```
Comment (jnthn) [inline lib/Agrammon/DataSource/Util.pm6:4]: Or `.return without $value;`.

File: lib/Agrammon/Web/Service.pm6 [hunk 3]
```diff
@@ -350,6 +350,7 @@ class Agrammon::Web::Service {
         my $results = self!get-outputs($user, $dataset-name)<results>;
         my $validation-errors = self!get-outputs($user, $dataset-name)<validation-errors>;
         warn '**** Got ' ~  $validation-errors.elems ~ ' input validation errors' if $validation-errors;
+        dd 'Validation errors:', $validation-errors if $validation-errors;
```
Comment (jnthn) [inline lib/Agrammon/Web/Service.pm6:353]: Debug leftover?

File: frontend/source/translation/fr.po (hunk 1 above)
Comment (zaucker) [reply frontend/source/translation/fr.po:980]: removed

File: lib/Agrammon/DataSource/Util.pm6 (hunk 2 above)
Comment (zaucker) [reply lib/Agrammon/DataSource/Util.pm6:4]: Neat. Done.

File: lib/Agrammon/Web/Service.pm6 (hunk 3 above)
Comment (zaucker) [reply lib/Agrammon/Web/Service.pm6]: No, on purpose for the moment. I think it is useful to have those in the log file on the server (see also comment on this PR at the top).
Added a comment.

**Review by jnthn — APPROVED** [review]

**Pull request merged**